rclcpp_lifecycle/README.md

```diff
@@ -0,0 +1,7 @@
+# `rclcpp_lifecycle`
+
+Package containing a prototype for lifecycle implementation.
```

Comment:
Add link to design documentation https://design.ros2.org/articles/node_lifecycle.html